docs/admin/troubleshooting.md

```diff
@@ -414,6 +414,51 @@ Verify if you're using Infiniband for `fabric_iface`: in the server config. The
 		$ dmg pool destroy --pool=$DAOS_POOL --force
 		Pool-destroy command succeeded
 
+### pmempool
+The pmempool is a management tool for Persistent Memory pool files created by PMDK libraries.
+DAOS uses the PMDK library to manage persistence inside ext4 files.
+[pmempool](https://pmem.io/pmdk/manpages/linux/v1.9/pmempool/pmempool-check.1/) can check consistency of a given pool file.
+It can be run with -r (repair) option which can fix some of the issues with pool file. DAOS will have more number of such pool file (vos-*), based
+on number of targets mention per daos engine. User may need to check each vos pool file for corruption on faulty pool.
+
```

Comment:
i would strongly advise to add a statement like "this tool should only be used under the direction of the DAOS support organisation" ... this sounds to me like there's a high probability to corrupt your DAOS system by trying to fix issues on this low-level PMem level without understanding the complete end-to-end view. or am i overly pessimistic here and the tool is always safe to run?

Reply:
This tool is part of PMDK so hopefully it should be safe but I don't have any evidence that this is 100% full proof or not. May be I can look at the PMDK side code and testing aspects of this tool and get more insight. But I agree that this can be dangerous if something does not go as per planned and data might be more screwed. So running under supervision will help not to loose the data. May be we have option to take backup first and try recovery on the original copy.

In general, all data recovery tools needs to be followed after the DAOS catastrophic recovery or may be prior. We need to try different scenarios and understand this tool recovery options and impact on DAOS at different level (Pool/Container/Objects). But adding warning will not harm at this stage.